src/views/WellnessCheck/components/HealthStatus/index.js

```diff
@@ -70,6 +70,19 @@ const HealthStatus = ({ currentStatus, setCurrentStatus, username, image }) => {
                 alt={username}
               />
             </Card>
+            {currentStatus === StatusColors.RED && (
```

Comment:
Yes, this will show the message to anyone who has a red status. If we had more time, we could get a more elegant solution where only people who haven't yet filled out the form would be shown the message. However, that requires building an extra integration for 360 to be aware of who that is. Instead, the plan is to set all student's status to red with no expiration, and then set up a one-way integration that will clear their status when they fill out the form. 

That way, the only people who will see the message without needing to fill out the form will be those with a red status for another reason. Presumably, that will be a small subset of people, since the red status is reserved for confirmed cases of exposure or positive tests, etc.

It's not the most elegant solution, but it works well and since the message is informative but not disruptive I think it's an acceptable compromise between functionality and speed.